fastlane/lib/fastlane/plugins/plugin_manager.rb

```diff
@@ -276,7 +273,7 @@ def self.load_plugins
         end
       end
 
-      if !loaded_plugins && PluginManager.new.pluginfile_content.to_s.include?(plugin_prefix)
+      if !loaded_plugins && Fastlane.plugin_manager.pluginfile_content.to_s.include?(PluginManager.plugin_prefix)
```

Comment:
No need to refer back out to the global variable here, right? We're already inside the only expected instance of the class?